archunit/src/main/java/com/tngtech/archunit/lang/syntax/elements/ClassesShould.java

```diff
@@ -411,16 +417,23 @@
      *
      * @param type An interface imported classes should NOT implement
      * @return A syntax element that can either be used as working rule, or to continue specifying a more complex rule
+     *
+     * @see ArchConditions#implement(Class)
```

Comment:
Shouldn't we consequently link for `ArchConditions#notImplement` from the `notImplement` methods (here and below)?
```suggestion
     * @see ArchConditions#notImplement(Class)
```
(Same for `notBeAssignable`{`To`,`From`}.)

Reply:
I thought there was only the positive case in `ArchConditions`, sure :+1: